goicicle/setup.sh

```diff
@@ -1,10 +1,24 @@
 #!/bin/bash
```

Comment:
This comment may be out of this PR's scope, so it's ok to ignore this comment.

We export the env var only. We don't persist it.

maybe we should add smtn like this:
```
echo "To make the library paths permanent, add the following lines to your ~/.bashrc or equivalent profile file:"
echo "export LD_LIBRARY_PATH=${LD_LIBRARY_PATH}:$TARGET_BLS12_377_PATH"
```

or just write it to bashrc ?

Reply:
I think running the script with source and sudo persists the path

Reply:
I think once you close that terminal or shell session, any changes made by the source are lost.

Reply:
These env vars are session specific, in order to make them permanent we will need to copy/move the shared lib files to a predefined location in the user/OS scope and then update bashrc/profile or /etc/init.d